domain/src/test/java/camp/nextstep/edu/domain/calculator/ExpressionParserTest.kt

```diff
@@ -0,0 +1,99 @@
+package camp.nextstep.edu.domain.calculator
+
+import org.junit.Assert.*
+import org.junit.Test
+
+
+class ExpressionParserTest {
+
+    private val expressionParser = ExpressionParser()
+
+
+    @Test
+    fun `1 + 1`() {
+        val expression = "1 + 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Plus, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 - 1`() {
+        val expression = "1 - 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Minus, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 * 1`() {
+        val expression = "1 * 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Multiply, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `1 나누기 1`() {
+        val expression = "1 / 1"
+        val parsed = expressionParser.parse(expression)
+        assertEquals(
+            listOf(Num(1), Operators.Divide, Num(1)),
+            parsed
+        )
+    }
+
+    @Test
+    fun `공백은 입력할 수 없다_0`() {
+        val expression = ""
+        assertThrows(IllegalArgumentException::class.java) {
+            expressionParser.parse(expression)
+        }
+    }
+
+    @Test
+    fun `공백은 입력할 수 없다_1`() {
```

Comment:
```suggestion
    fun `공백 문자열을 입력하는 경우 IllegalArgumentException이 발생한다`() {
```

위와 마찬가지로 함수명이 직관적으로 변경하는 것이 좋아보여요..!